modules/freetype/SCsub

```diff
@@ -67,8 +67,6 @@ if env["builtin_freetype"]:
     env.Prepend(CPPPATH=[thirdparty_dir + "/include"])
 
     env_freetype.Append(CPPDEFINES=["FT2_BUILD_LIBRARY", "FT_CONFIG_OPTION_USE_PNG", "FT_CONFIG_OPTION_SYSTEM_ZLIB"])
-    if env.dev_build:
-        env_freetype.Append(CPPDEFINES=["ZLIB_DEBUG"])
```

Comment:
I removed this as we don't build zlib anymore in `env_freetype`.

This could still affect the zlib *headers* included by freetype files, but I'm not sure how their debug features would work if the `.c` files were compiled without that define. Overall I don't think we care to dig into zlib debug internals in our "normal" dev builds, so I don't think we'll miss it.

Reply:
We should make sure that these match if we use the vendored zlib.

Reply:
Right, we do have this in `core/SCsub`:
```
    if env.dev_build:
        env_thirdparty.Append(CPPDEFINES=["ZLIB_DEBUG"])
```

So this should be `if env.dev_build and env["builtin_zlib"]`.

Reply:
Done in the latest commit.